src/design-tokens/tier-3-component/focus-ring.json

```diff
@@ -10,6 +10,18 @@
             "value": "{eds.color.neutral.white}"
           }
         }
+      },
+      "box": {
+        "focus-ring": {
+          "outline": {
+            "width": {
+              "value": "{eds.outline.width.sm}"
```

Comment:
by using a static definition with the `{...}` syntax, if someone redefines the tier one values, they would also need to update these two tokens. If we use the css `var` syntax, overriding the base value(s) will propagate to these due to cascading

Reply:
Will switch to the css custom property

Reply:
we can put that on the table for consideration. I think we will probably want to use references in many/most cases, but there may be unthemed things where we should just use the static references. That can work as a brief heuristic for now until we consider some pros/cons.